src/containers/Articles/Article/RatingContainer.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/prefer-default-export */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import Rating from '../../../components/Rating';
+import { rateFetch, updateRateValue } from '../../../store/actions/async/rate';
+import { infoToast, successToast } from '../../../components/UI/Toast/toast';
+
+
+export class RatingContainer extends Component {
+    state = {
+        rate_score: 0
+    }
+    componentDidMount = () => {
+        const { slug } = this.props.match.params;
+        const {
+            history,
+            rateFetch
+        } = this.props;
+        const url =
+            `https://ah-backend-summer-staging.herokuapp.com/api/v1/articles/${slug}`;
+        const payload = {
+            url,
+            history: history
+        };
+        rateFetch(payload);
+    }
+
+    onStarClick = (nextValue) => {
+        const articleSlug = this.props.match.params.slug;
+        const url =
+            `https://ah-backend-summer-staging.herokuapp.com/api/v1/articles/${articleSlug}/rate`;
+        const {
+            history,
+            rate
+        } = this.props;
+        this.setState({ rate_score: nextValue });
+        const data = { rate_score: nextValue };
+
+        const payload = {
+            data,
+            url,
+            history: history,
+        };
+        rate(payload);
+    }
+
+    render() {
+        const { rate_score } = this.state;
+        const ratingProps = {
+            rate_score: rate_score,
+            onStarClick: this.onStarClick,
+            average_rating: this.props.average_rating
+        };
+        this.rateRender = (
+            <Rating {...ratingProps} />
+        );
+
+        
+        if (this.props.error !== null){
```

Comment:
`!== null` is unnecessary. The same issue on line 68. 
We can also use destructuring to extract data on props.